deps/chakrashim/include/v8.h

```diff
@@ -2736,8 +2737,10 @@ class V8_EXPORT TryCatch {
   void SetNonUser() { user = false; }
   void GetAndClearException();
   void CheckReportExternalException();
+  JsValueRef EnsureException() const;
```

Comment:
What are thoughts on this? The `const` is a bold faced lie, but some of the methods that call it are `const` and the `const_cast` was there before.